docker_explorer/storage.py

```diff
@@ -79,36 +77,21 @@ def _MakeExtraVolumeCommands(self, container_object, mount_dir):
         be mounted.
```

Comment:
this returns a list of lists  shouldn't it be `list(list(str))` in the return type?

Reply:
done